src/coreclr/jit/emit.h

```diff
@@ -607,6 +607,7 @@ class emitter
         unsigned          idNum;
         size_t            idSize;        // size of the instruction descriptor
         unsigned          idVarRefOffs;  // IL offset for LclVar reference
+        unsigned          idVarRefOffs2; // IL offset for 2nd LclVar reference (in case this is a pair)
```

Comment:
Are these offsets actually maintained/set/used enough to care about them? Last time I checked the logic, that was not the case.